cmd/titus-imds-cni/env.go

```diff
@@ -188,36 +214,37 @@ func openFile(dname, fname string) (*os.File, error) {
 	return fd, nil
 }
 
-func writeContainerInfo(pod *v1.Pod) error {
-	str, ok := pod.Annotations["containerInfo"]
+func writeContainerInfo(pod *v1.Pod, env map[string]string) error {
+	ipv4Addr, ok := env["EC2_LOCAL_IPV4"]
```

Comment:
I count 4 references in non-test code to this string.  Maybe time for a constant, but a different PR might be better.